gnmpagerduty/tests/test_tasks.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import
+import unittest
+
+
+class TestHumanFriendly(unittest.TestCase):
+    def test_should_match(self):
+        from tasks import human_friendly
+
+        self.assertEqual(human_friendly(8589934592),"8.0GiB")
+        self.assertEqual(human_friendly(17592186044416),"16.0TiB")
+        self.assertEqual(human_friendly(28037546508288),"25.5TiB")
+
```

Comment:
every piece of your timed task function, that checks the storages and sends to pagerduty.  Use mocks to check function calls to stuff like vidispine, pd, etc.